web/src/app/modules/shared/components/presentation/indicator/indicator.component.html

```diff
@@ -1,18 +1,14 @@
-<ng-container *ngIf="currentStatus">
-  <ng-container *ngIf="detail !== undefined; else showStatus">
+<ng-container *ngIf="status && iconShape">
```

Comment:
Not a huge deal. `status` can be replaced with `currentStatus` as both `currentStatus` and `iconShape` are calculated.